e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -99,26 +129,28 @@ describe("snapshots", () => {
     });
   }
 
-  function addUsersAndGroups() {
+  function addUsersAndGroups(isEE = false) {
+    const lowest_read_data_permission = isEE ? "blocked" : "unrestricted";
+
     // groups
     cy.request("POST", "/api/permissions/group", { name: "collection" }).then(
       ({ body }) => {
-        expect(body.id).to.eq(COLLECTION_GROUP); // 4
+        expect(body.id).to.eq(COLLECTION_GROUP); // 3
```

Comment:
Just correcting these comments - e.g. `COLLECTION_GROUP` is set to `3` in `cypress_data.js`.